tests/test_judgment.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+from nose.tools import *  # noqa
+from datetime import date
+
+from cobalt.judgment import Judgment
+from cobalt.akn import datestring
+
+
+class JudgmentTestCase(TestCase):
```

Comment:
A lot of this functionality comes from `StructuredDocument`, so I'd argue that these tests could move to a `StructuredDocumentTestCase` class, and not worry about it here. Otherwise we're going to be duplicating test cases for every document type. It's better to test the class on which the functionality is declared.